src/metabase/public_settings/premium_features.clj

```diff
@@ -21,16 +21,21 @@
   "Schema for a valid premium token. Must be 64 lower-case hex characters."
   #"^[0-9a-f]{64}$")
 
-(def store-url
-  "URL to the MetaStore. Hardcoded by default but for development purposes you can use a local server. Specify the env
-  var `METASTORE_DEV_SERVER_URL`."
+(def token-check-url
+  "Base URL to use for token checks. Hardcoded by default but for development purposes you can use a local server.
+  Specify the env var `METASTORE_DEV_SERVER_URL`."
   (or
-   ;; only enable the changing the store url during dev because we don't want people switching it out in production!
+   ;; only enable the changing the token check url during dev because we don't want people switching it out in production!
    (when config/is-dev?
      (some-> (env :metastore-dev-server-url)
              ;; remove trailing slashes
              (str/replace  #"/$" "")))
-   "https://store.metabase.com"))
+   "https://token-check.metabase.com"))
+
+(def store-url
+  "Store URL, used as a fallback for token checks"
```

Comment:
This isn't only used for token checks, it looks like it's also used for `cloud-gateway-ips-url`. Probably worth mentioning that here